box/boxunreal.py

```diff
@@ -28,9 +30,15 @@ def __init__(
         if self.dataset_path:
             self.dataset_path.mkdir(parents=True, exist_ok=True)
 
+        self.navigator.set_forward_increment(forward_increment)
+        self.raycast_length = forward_increment + 10.0
```

Comment:
Why `+ 10.0`?

Reply:
This was also a part of the debugging process. It should not actually be included. Thanks for bringing it to my attention.